OCABundles/schema/bcgov-digital-trust/person/OCABundle.json

```diff
@@ -144,34 +144,34 @@
       },
       {
         "capture_base": "ES3khN7HC1zZyr-DeQ6UbUVyhY_npOrxX_8NO_K4IWH8",
-        "credential_help_text": "Example credential help text.",
+        "credential_help_text": "",
         "credential_support_url": "https://www.gov.bc.ca/",
         "description": "A Canadian schema used for the identity of a person",
         "digest": "EFqwZey67Xkfl4q_rcYroNzZa0f8SUdIdRLAKf4kFU_U",
         "issuer": "Government of British Columbia",
         "issuer_description": "Government of British Columbia",
         "issuer_url": "https://www.gov.bc.ca/",
-        "language": "en-CA",
+        "language": "en",
         "name": "Person Credential",
         "type": "spec/overlays/meta/1.0"
       },
       {
         "capture_base": "ES3khN7HC1zZyr-DeQ6UbUVyhY_npOrxX_8NO_K4IWH8",
-        "credential_help_text": "Exemple de texte d'aide sur les informations d'identification.",
+        "credential_help_text": "",
         "credential_support_url": "https://www.gov.bc.ca/",
         "description": "Schéma canadien utilisé pour l'identité d'une personne",
         "digest": "E5FewgxojVholk4-HjrIbz9iyYqS-B3NOYUVmo5VkFYc",
         "issuer": "Gouvernement de la Colombie-Britannique",
         "issuer_description": "Gouvernement de la Colombie-Britannique",
         "issuer_url": "https://www.gov.bc.ca/",
-        "language": "fr-CA",
+        "language": "fr",
         "name": "Informations d'identification de la personne",
         "type": "spec/overlays/meta/1.0"
       },
       {
-        "logo": "https://raw.githubusercontent.com/bcgov/bc-wallet-mobile/main/app/src/assets/branding/bc-logo.jpg",
+        "logo": "https://raw.githubusercontent.com/bcgov/aries-oca-bundles/main/OC/bc-logo.jpgema/bcgov-digital-trust/person/bc-logo.png",
```

Comment:
This path looks a bit odd

Reply:
good catch, copying error, I've fixed it now